_pages/governance.md

```diff
@@ -63,23 +62,24 @@ computing field.
 
 The project is a community-driven effort.
 BIDS, originally OBIDS, was initiated during an INCF sponsored data
-sharing working group meeting (January 2015) at Stanford University. It
-was subsequently spearheaded and maintained by Chris Gorgolewski. The
-project is currently managed and maintained by Franklin Feingold, Stefan
-Appelhoff, and the Poldrack Lab at Stanford. BIDS has advanced under the
-direction and effort of contributors, the community of researchers that
-appreciate the value of standardizing neuroimaging data to facilitate
-sharing and analysis. The project is multifaceted, and depends on
-contributors for: specification development and maintenance, [BIDS
-Extension Proposals
-(BEPs)](/get_involved), software tools, [starter
-kits](https://bids-standard.github.io/bids-starter-kit/){:target="_blank"},
-[examples](https://github.com/bids-standard/bids-examples){:target="_blank"}, and general
-discussions. The relevant discussions are located in our [Google
-Group](https://groups.google.com/forum/#!forum/bids-discussion){:target="_blank"}, [GitHub
-organization](https://github.com/bids-standard){:target="_blank"}, and public Google
-Documents (typically associated with an [extension
-proposal](/get_involved)).
+sharing working group meeting (January 2015) at Stanford University.
+It was subsequently spearheaded and maintained by Chris Gorgolewski.
+In the transitional period after Chris Gorgolewski's departure in early 2019
+and before the community's acceptance of the present governance in late 2019,
+the project was managed and maintained by Franklin Feingold, Stefan Appelhoff, and the Poldrack Lab at Stanford.
```

Comment:
This is the only part where I added / rephrased. The remaining changes are line breaks.